### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by tzufgoogle**
Added README.md files for generating the RVS and Wikigeo.

### CODE REVIEW

File: cabby/data/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+To create an OSM-based sample we query OSM and extract POI that are not large (spread over less than 10 s2cellids of level 18.), within region, and have the following tag keys: 'name', 'amenity', 'colour', 'brand', 'tourism', 'leisure', 'historic', 'building', 'description', 'building:colour', 'building:material', 'roof:material', 'roof:shape', 'roof:colour'.  Like the sample Wikidata-based samples, we concatenate the tag values into one semi-sentence. E.g,  “St. Paul Cathedral Rectory and building and Dithridge Street”.
```
Comment (jasonbaldridge) [inline cabby/data/README.md:12]: Please make this a list.

File: cabby/geo/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generating spatial samples
+
+## Overview
+To create spatial samples 3 steps should be taken: (1) pick a path (start and end points and a route); (2) pick landmarks; (3)  calculate spatial-features such as cardinal direction.
+
+### (1) Picking a path
+To construct a path we first need to pick the start and end point. We start by selecting the end point. We randomly select a POI from the OSM-based  graph constructed, that is a small POI and has a tag key of a name. We define a small POI if its geometry covers 4 or less S2Cells of level 18 (no more than 152 meters long). We then pick the start point from the POI in the OSM-based graph based on the following conditions: (1)  a small POI; (2) within a path distance of 200-2000 meters from the end point; (3) has a tag key of name. We then pick a route that is the shortest distance between start and end point.
```
Comment (jasonbaldridge) [inline cabby/geo/README.md:7]: Please make 1-3 a list.

File: cabby/rvs/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+bazel-bin/cabby/geo/map_processing/map_processor --region REGION --min_s2_level LEVEL --directory DIRECTORY_TO_MAP
+```
+### (2) Generating spatial samples
+```
+bazel-bin/cabby/geo/sample_poi --region REGION --min_s2_level LEVEL --directory DIRECTORY_TO_MAP --path PATH_TO_SPATIAL_ITEMS.gpkg --n_samples NUMBER_OF_SAMPLES_TO_GENERATE
+```
+
+### (5) Generating RVS instructions
+```
+bazel-bin/cabby/rvs/generate_rvs --rvs_data_path PATH_TO_SPATIAL_ITEMS.gpkg --save_instruction_path PATH_TO_RVS.json
```
Comment (jasonbaldridge) [inline cabby/rvs/README.md:20]: Have you verified that these commands work with the latest code? (E.g. 'path' became 'output_dir')

File: cabby/data/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# WikiGeo - a dataset to support the geolocation subtask
```
[the remark this answers is not in the document]
Comment (tzufgoogle) [reply cabby/data/README.md:1]: Added Wikipedia exanples (OSM and Wikidata were already there)

File: cabby/data/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+To create a Wikipedia backlinks sample we use the Wikipedia pages extracted.  We query Wikimedia API for Wikipedia backlinks to the Wikipedia page extracted in 1. The backlinks are Wikipedia  pages that have some link to the Wikipedia pages queried on. For each sentence in the backlink with a mention of the entity queried on we extract it and construct a WIkiGeo sample.
+
+To create a Wikidata-based sample we query Wikidata by region and extract properties found: (a) The place name (b) the place description (c) architectural style (d) use (e.g., hotel) (e) has part (e.g., Knoxville Branch Library is part of Carnegie Library of Pittsburgh) (f) subsidiary (e.g., the subsidiary of CMU is Carnegie Mellon Silicon Valley) (g) alternative name (h) instance (e.g., Hillman Library is an instance of a  library). We concatenate all these tags into one semi-sentence which is one sample in the WikiGeo dataset.  E.g, “Heinz Field and American football stadium that is home to the Pittsburgh Steelers and Pittsburgh Panthers and Steelers Stadium and stadium.”
```
[the remark this answers is not in the document]
Comment (tzufgoogle) [reply cabby/data/README.md]: Yes. done.

File: cabby/data/README.md (hunk 1 above)
Comment (tzufgoogle) [reply cabby/data/README.md]: Done

File: cabby/geo/README.md (hunk 2 above)
Comment (tzufgoogle) [reply cabby/geo/README.md]: Done

File: cabby/rvs/README.md (hunk 3 above)
Comment (tzufgoogle) [reply cabby/rvs/README.md]: It works. I ran it with DC region level 18 and generated 1 sample.

**Review by tzufgoogle — COMMENTED** [review]
Added lists.

**Review by jasonbaldridge — APPROVED** [review]
LGTM, thanks!

**Pull request merged**